scalafix-core/shared/src/main/scala/scalafix/internal/util/TypeSyntax.scala

```diff
@@ -1,138 +1,451 @@
 package scalafix.internal.util
 
+import java.io.PrintWriter
+import java.io.StringWriter
+import org.langmeta.internal.semanticdb._
 import scala.meta._
+import scala.meta.internal.{semanticdb3 => s}
 import scala.reflect.ClassTag
+import scala.util.control.NonFatal
 import scalafix._
 import scalafix.internal.util.SymbolOps.SymbolType
-import scalafix.util.SymbolMatcher
-import scalafix.util.TreeExtractors._
 import org.langmeta.semanticdb.Symbol
+import scala.meta.internal.semanticdb3.SingletonType.{Tag => x}
```

Comment:
This saves two characters and makes the code harder to read.  Why not just Tag.?

Reply:
I've replaced `x` with `Tag` but kept the import local, `Tag` is used in `s.Type.Tag`/`s.SingletonType.Tag` and a few others I think.